src/program/lwaftr/tests/subcommands/config_test.py

```diff
@@ -0,0 +1,228 @@
+"""
+Test the "snabb lwaftr config" subcommand. Does not need NIC names because
+it uses the "bench" subcommand.
+"""
+
+import json
+import os
+from signal import SIGTERM
+import socket
+from subprocess import PIPE, Popen
+import time
+import unittest
+
+from lib.test_env import BENCHDATA_DIR, DATA_DIR, ENC, SNABB_CMD, BaseTestCase
+
+
+DAEMON_PROC_NAME = 'config_test_daemon'
+DAEMON_ARGS = (
+    str(SNABB_CMD), 'lwaftr', 'bench', '--reconfigurable',
+    '--bench-file', '/dev/null',
+    '--name', DAEMON_PROC_NAME,
+    str(DATA_DIR / 'icmp_on_fail.conf'),
+    str(BENCHDATA_DIR / 'ipv4-0550.pcap'),
+    str(BENCHDATA_DIR / 'ipv6-0550.pcap'),
+)
+SOCKET_PATH = '/tmp/snabb-lwaftr-listen-sock-%s' % DAEMON_PROC_NAME
+
+
+class TestConfigGet(BaseTestCase):
+    """
+    Test querying from a known config, testing basic "getting".
+    It performs numerous gets on different paths.
+    """
+
+    daemon_args = DAEMON_ARGS
+    wait_for_daemon_startup = True
+
+    config_args = (str(SNABB_CMD), 'config', 'get', DAEMON_PROC_NAME)
+
+    def test_get_internal_iface(self):
+        cmd_args = list(self.config_args)
+        cmd_args.append('/softwire-config/internal-interface/ip')
+        output = self.run_cmd(cmd_args)
+        self.assertEqual(output.strip(), b'8:9:a:b:c:d:e:f',
+            '\n'.join(('OUTPUT', str(output, ENC))))
```

Comment:
PEP-8 allows for hanging indentation, and as agreed on Matrix, I'll switch to that, rather than using this style that I strongly dislike.